tools/validate_configs/validate_configs.sh

```diff
@@ -21,16 +21,28 @@ run_test() {
 	exampleFile=$(basename "$example")
 	DEPLOYMENT=$(echo "${exampleFile%.yaml}-$(basename "${tmpdir##*.}")" | sed -e 's/\(.*\)/\L\1/')
 	PROJECT="invalid-project"
+	GHPC_PATH="${cwd}/ghpc"
+	# Cover the three possible starting sequences for local sources: ./ ../ /
+	LOCAL_SOURCE_PATTERN='source:\s\+\(\./\|\.\./\|/\)'
```

Comment:
nit, do not resubmit because of this, but I think you could have made it simpler with something like:
'source:\s\+\.\{0,2\}/'
I am never too sure of the different regexp syntaxes.. I *think* this one should hold.